app/controllers/articles_controller.rb

```diff
@@ -1,4 +1,5 @@
 class ArticlesController < ApplicationController
+  before_action :find_article
```

Comment:
Scope it to the edit, show, edit and destroy actions. There's a risk that you end up with ActiveRecord::RecordNotFound error if you don't 
```
before_action :find_article, only: [:show, :show, :destroy, :edit]
```
or the slightly shorter:
```
before_action :find_article, except: [:create]
```